webapi/src/main/scala/org/knora/webapi/store/triplestore/upgrade/RepositoryUpdater.scala

```diff
@@ -44,41 +39,22 @@ final case class RepositoryUpdater(triplestoreService: TriplestoreService) {
    */
   val maybeUpgradeRepository: Task[RepositoryUpdatedResponse] =
     for {
-      foundRepositoryVersion    <- getRepositoryVersion()
-      requiredRepositoryVersion <- ZIO.succeed(org.knora.webapi.KnoraBaseVersion)
-
-      // Is the repository up to date?
-      repositoryUpToDate <- ZIO.succeed(foundRepositoryVersion.contains(requiredRepositoryVersion))
-
+      repositoryVersion <- getRepositoryVersion
       repositoryUpdatedResponse <-
-        if (repositoryUpToDate) {
-          // Yes. Nothing more to do.
-          ZIO.succeed(RepositoryUpdatedResponse(s"Repository is up to date at $requiredRepositoryVersion"))
-        } else {
+        if (repositoryVersion.contains(KnoraBaseVersion)) {
```

Comment:
shouldn't it check that it's strictly smaller? or are you anyway relying on the fact that only the plugins that are bigger are filtered?

Reply:
Yes, I think it would make most sense to not do anything, but maybe since it's just the built in graph updates, it should be fine.

Reply:
This happens regularly on dev servers, when the API version deployed there is lower than the one on prod, and then prod data gets mirrored to that dev server.

And previously, in this scenario the start-up would fail because of the exception raised in L139 which killed the fiber. I think we should keep that behaviour, unless we have good reasons to change it.

Reply:
This commit add the behaviour back again: 
https://github.com/dasch-swiss/dsp-api/pull/3216/commits/bf97820d833ba44e39ef74a9ee2ed278d4e93e4f